charm.go

```diff
@@ -38,3 +40,29 @@ func ReadCharm(path string) (charm Charm, err error) {
 	}
 	return charm, nil
 }
+
+// SeriesToUse takes a specified series and a list of series supported by a
+// charm and returns the series which is relevant.
+// If the specified series is empty, the any default series defined by the
```

Comment:
"any" needs to be read as "any default"
If the charm does not specify series, the default series is ""
The default is the first series specified, if any are specified.